examples/go/with_proto/go/lib/lib_test.go

```diff
@@ -0,0 +1,18 @@
+package lib
+
+import "testing"
+
+func TestMyFun(t *testing.T) {
+	t.Run("3 + 2 = 5", func(t *testing.T) {
```

Comment:
I think the plugin currently does not support running a single `t.Run` instead it runs the whole defining `func`. Can we add another `func`, i.e. at TestMyFun level, to test this?